File: CursoPOOUber/PHP/card.php
<?php
    require_once('payment.php');
    class Card extends Payment {
        public $number;
        public $cvv;
        public $date;

        public function __construct($mount, $number, $cvv, $date){
            parent::__construct($mount);
            $this->number = $number;
            $this->cvv = $cvv;
            $this->date = $date;
        }

        public function printDataCard(){
            echo "Number: $this->number , CVV:  $this->cvv , Date: $this->date";
        }
    }
?>

File: CursoPOOUber/Python/card.py
from datetime import datetime
from payment import Payment


class Card(Payment):
    number = float
    cvv = int
    date = datetime
    def __init__(self, mount, number, cvv, date):
        super().__init__(mount)
        self.number = number
        self.cvv = cvv
        self.date = date